index6.js

```diff
@@ -0,0 +1,24 @@
+
+// const names =(arr, nam)=>{
+//     return arr.find((user, index)=>{
+
+//        return user === nam ? `Пользователь с именем ${nam} найден!Его индекс User ${index}` : `Пользователь не найден`;
+//     })
+// }
+
+
+
+function names(arr, elem) {
+    for (var i = 0; i < arr.length; i++) {
+        if (arr[i] === elem) {
+            return `Пользователь с именем ${elem} найден!Его позиция - ${i}`;
+        }
+    }
+    return `Пользователь не найден!`;
+}
+
```

Comment:
![image](https://user-images.githubusercontent.com/55699257/96995099-55fd1a00-1536-11eb-80a0-720f233f9d91.png)
Google is good, but this solution has the list of problems:
1) Never use `var` !
2)Never use `for` !

Rewrite this function by your own implementation please :)